src/components/screens/editor/draft-editor-screen.tsx

```diff
@@ -334,6 +335,12 @@ const DraftEditorScreen: React.FC = () => {
           </RoundActionButton>
         </span>
       </Tooltip>
+      <RoundActionButton
+        onClick={() => FileUtils.exportToZip(draftForm)}
+        startIcon={<GetApp/>}
+      >
+        <Typography>{ strings.draftEditorScreen.exportToZip }</Typography>
+      </RoundActionButton>
```

Comment:
Same here, this should be visible only with specific role. Same as the "scripts"/extensions tab